generator-core/pom.xml

```diff
@@ -5,24 +5,21 @@
     <parent>
         <artifactId>spring-boot-cli</artifactId>
         <groupId>br.com.netodevel</groupId>
-        <version>0.0.1.BUILD-SNAPSHOT</version>
+        <version>0.0.2.BUILD-SNAPSHOT</version>
```

Comment:
Consider using a more specific versioning scheme, as `BUILD-SNAPSHOT` might not be suitable for production environments. For example, you could use a timestamp or a build number instead.